data/core/encyclopedia/calendar.cfg

```diff
@@ -25,6 +23,10 @@ Stillseed
 Reaper’s Moon
 Verglas Bloom
 Blackfire
+Whitefire
+Bleeding Moon
+
+The Wesnoth calendar begins with the founding of Wesnoth in the year 1 YW (short for Years Wesnoth. Years before that are counted based on the number of years before the founding of Wesnoth using BW (short for Before Wesnoth.) After the fall of Wesnoth dates use AF, short for after the fall.
```

Comment:
It's a bit inconsistent that you capitalized "Years Wesnoth" and "Before Wesnoth" but not "after the fall". I'd also say that "Year of Wesnoth" is a better expansion of YW.